source/MRMesh/MRTriangleIntersection.cpp

```diff
@@ -1,6 +1,41 @@
 #include "MRTriangleIntersection.h"
 #include "MRGTest.h"
 
+namespace
+{
+    constexpr MR::Triangle3f triangle1 {
+        MR::Vector3f { -30.f, -52.f, 0.f },
+        MR::Vector3f { +30.f, -52.f, 0.f },
+        MR::Vector3f { 0.f, 0.f, 0.f },
+    };
+    constexpr MR::Vector3f add( const MR::Vector3f& a, const MR::Vector3f& b )
```

Comment:
let us remove this func, and use operator +

Reply:
Fixed.